integration/bundle/python_wheel_test.go

```diff
@@ -47,14 +47,6 @@ func runPythonWheelTest(t *testing.T, templateName, sparkVersion string, pythonW
 	require.Contains(t, out, "['my_test_code', 'param1', 'param2']")
 }
 
-func TestPythonWheelTaskDeployAndRunWithoutWrapper(t *testing.T) {
-	runPythonWheelTest(t, "python_wheel_task", "13.3.x-snapshot-scala2.12", false)
-}
-
-func TestPythonWheelTaskDeployAndRunWithWrapper(t *testing.T) {
-	runPythonWheelTest(t, "python_wheel_task", "12.2.x-scala2.12", true)
```

Comment:
@andrewnester why was this Spark version used in this test?

Reply:
Thanks! Added you comment into the test 060b8f50898fdc4fd1032f75e8fc0490c6221f49